sec_certs/utils/pandas.py

```diff
@@ -9,7 +9,7 @@
 from dataclasses import dataclass
 from pathlib import Path
 from shutil import copyfile
-from typing import Any, Dict, Final, List, Optional, Set, Tuple, Union
+from typing import Any, Final
```

Comment:
This file does not have the annotations import from the future yet it was transformed to the Python >= 3.9 form of annotations. Was this manual or some pyupgrade issue? In any way, the import needs to be added otherwise this will likely break on Python 3.8.